src/test/perl/lock.t

```diff
@@ -2,49 +2,26 @@
 use strict;
 use warnings;
 use Test::More;
-use CAF::Lock qw(FORCE_IF_STALE FORCE_ALWAYS);
+use CAF::Lock qw(FORCE_ALWAYS);
 
 use constant LOCK_TEST_DIR => "target/tests";
 use constant LOCK_TEST => LOCK_TEST_DIR . "/lock-caf";
 
 mkdir(LOCK_TEST_DIR);
 unlink(LOCK_TEST);
 
-my $lock=CAF::Lock->new(LOCK_TEST);
+my $lock1 = CAF::Lock->new(LOCK_TEST);
+my $lock2 = CAF::Lock->new(LOCK_TEST);
 
-ok(!$lock->is_locked(), "Unlocked at start");
-my $lockpid=$lock->get_lock_pid();
+ok(!$lock1->is_set(), "Unlocked at start");
+ok($lock1->set_lock(), "Lock set");
```

Comment:
the main reason for these extra tests is to make sure that instances do not share any states. if lock1 takes the lock, lock2 should be unmodified